### PR TITLE
#28 First pass, README edits

### PR DESCRIPTION
**Pull request by tsyroid**
Fixes: #28 

Changes we discussed over a week ago. First pass, more to become familiar with the collaboration process than actually change shit 😎 .

Review using `:MarkdownPreview` as that's what I used to verify changes. Note I used strikethrough in a couple places; ie, I did not actually delete words, but indicated some that 'might' want to go away.

Also be sure to advise on things like:
- how I worded commit message
- branch name? too long?
- the way I'm using this fork/submit pull request process
- anything workflow related you want to see me do different

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -329,10 +330,12 @@ function()
   vim.lsp.buf.formatting_sync(nil, 1500)
 end
 ```
-
-If you need to call a function from a plugin, but do not want the require that the plugin be
-loaded at the time you define your keymaps (like if you're using Packer to lazy-load plugins),
-you can use the `lazy_required_fn` helper:
+<!-- don't like the wording 'don't require the plugin _be_ loaded
```
Comment (mrjones2014) [inline README.md:333]: how about "do not want to require that the plugin is loaded at the time you define your keymaps"

**Comment by mrjones2014** [comment]
These changes all look good to me. I'm fine with removing all the words you've ~~strikethroughed~~

**Comment by tsyroid** [comment]
OK, good. It’s in your hands then. And I’ll got back and revisit that
‘awkward’ bit later. Need to let my brain churn that one around a bit.

And how about workflow, commit message structure, etc? Your codebase is
remarkably clean, and I want to ensure if I’m contributing, I do all the
right things to keep it that way.

**Comment by mrjones2014** [comment]
>And how about workflow, commit message structure, etc?

All looks good to me :sunglasses:

File: README.md [hunk 2]
```diff
@@ -20,11 +20,11 @@ Define your keymaps, commands, and autocommands as simple Lua tables, building a
 
 ## Prerequisites
 
-- Neovim nightly (0.7+), specifically, this plugin depends on the following APIs:
+- Neovim nightly (0.7+); specifically, this plugin depends on the following APIs:
   - `vim.keymap.set`
   - `vim.api.nvim_create_augroup`
   - `vim.api.nvim_create_autocmd`
-- (Optional) A good `vim.ui.select()` handler, this is what provides the UI for the finder.
+- (Optional) A ~~good~~ `vim.ui.select()` handler; this ~~is what~~ provides the UI for the finder.
```
Comment (mrjones2014) [inline README.md:27]:
```suggestion
- (Optional) A `vim.ui.select()` handler; this provides the UI for the finder.
```

File: README.md [hunk 3]
```diff
@@ -329,10 +330,12 @@ function()
...
-
-If you need to call a function from a plugin, but do not want the require that the plugin be
-loaded at the time you define your keymaps (like if you're using Packer to lazy-load plugins),
-you can use the `lazy_required_fn` helper:
+<!-- don't like the wording 'don't require the plugin _be_ loaded
+  'be loaded' feels awkward
+  but I can't think of a good replacemet ATM -->
+If you need to call a function from Legendary, but don't require the plugin be loaded at the time
+you define your keymaps (for example, if you're using Packer to lazy-load plugins), you can use the
+`lazy_required_fn` helper:
```
Comment (mrjones2014) [inline README.md:338]:
```suggestion
If you need to call a function from Legendary, but the plugin won't be loaded at the time
you define your keymaps (for example, if you're using Packer to lazy-load plugins), you can use the
`lazy_required_fn` helper:
```

**Review by mrjones2014 — CHANGES_REQUESTED** [review]
@tsyroid let's just make these changes and I'll merge.

**Review by mrjones2014 — APPROVED** [review]

**Comment by tsyroid** [comment]
Agreed. Carry on.

I’ll be revisiting this file multiple times as I work through understanding
legendary and as I configure my version here.

**Pull request merged**